CHANGELOG.md

```diff
@@ -16,17 +16,25 @@
   [Muhammad Zeeshan](https://github.com/mzeeshanid)
   [#2802](https://github.com/realm/SwiftLint/issues/2802)
 
+* Add new `ignore_attributes` option to `redundant_type_annotation` rule
+  that allows disabling the rule for properties that are marked with at least
+  one of the configured attributes.
```

Comment:
Please append two spaces to the end of the description. They are required to enforce a line break in the rendered Markdown.